src/components/CustomRequestScreen/Near/NearSignAndSendTransaction.vue

```diff
@@ -0,0 +1,156 @@
+<script setup lang="ts">
+import Decimal from 'decimal.js'
+import { type SignAndSendTransactionOptions } from 'near-api-js/lib/account'
+import { computed } from 'vue'
+
+import SignMessageAdvancedInfo from '@/components/SignMessageAdvancedInfo.vue'
+import { estimatedNearTxFees } from '@/utils/near/estimatedTxFees'
+import { getActionName } from '@/utils/near/getActionName'
+import { getRequestHandler } from '@/utils/requestHandlerSingleton'
+import { truncateMid } from '@/utils/stringUtils'
+
+const props = defineProps<{
+  transaction: SignAndSendTransactionOptions
+}>()
+
+const accountHander = getRequestHandler().getAccountHandler()
+const nearDecimals = accountHander.decimals
+
+const estimatedTxFees = computed(() => {
+  const totalTxFees = props.transaction.actions.reduce((acc, action) => {
```

Comment:
Are you sure this won't overflow (you are adding Numbers) or become inexact (since NEAR fees are decimal and you are converting them to JS numbers)?

Reply:
I don't think this will overflow, since these are the known constants we are using from [utils/near/estimatedTxFees.ts](https://github.com/arcana-network/wallet-ui/pull/706/files/42625aed08d2ae690f11389252a2377180b2e05b#diff-c569624c49035f1086a2598cf2cc6ce3eb6c53c0273c25c3dd9e99f2fc597f93), and I don't think these static numbers will overflow on addition. Let me know your thoughts, if needed we can change this.

Btw this is just used for display purpose, we don't supply it anywhere during the transaction. Just a presentation layer stuff. Also I didn't find a better way to estimate tx fees (gas fees) in NEAR, so using constant values that I found in their docs